### ISSUE TITLE
aiven_influxdb was deleted during upgrade

### ISSUE THREAD

**Issue by YingwenX** [aiven/terraform-provider-aiven#1458]
## What can we help you with?

i'm using terraform aiven provider for influxdb creation, but i find out that during the update, the whole service will be deleted and recreated

## Where would you expect to find this information?
what i'm expecting is that during the upgrade the service including database will not be deleted, and just the upgrade

**Comment by Serpentiel** [comment]
Hey, @YingwenX! 👋 

Thank you for raising this question!

May I know exactly what actions are you performing? Also, an example of your manifest would help a lot, too!

**Comment by YingwenX** [comment]
yes, i just use aiven provider to create and update aiven service, and when i tried to update service name, the plan looks unlikely what i expacted, because it totally removed the defaultdb, i don't know why
please reach out to the attachment with apply changes to see details
[aiven_plan_db_removed.txt](https://github.com/aiven/terraform-provider-aiven/files/13522590/aiven_plan_db_removed.txt)

**Comment by Serpentiel** [comment]
Hey, @YingwenX! 👋

Currently Aiven API does not supports changing the name of the service, but it is definitely something that might be considered in future.

Since Terraform is a "smart API client", when you try to change the name of the service, it suggests to recreate it for you, which is expected because that's the only way you can technically change the name of the service right now, by fully recreating it.

I'm afraid this issue falls outside of the scope of the Terraform Provider. I will close this issue now, please let me know if there are any future comments, suggestions or ideas, or if I misunderstood your problem.
